extensions.json

```diff
@@ -54,13 +54,15 @@
     },
     {
       "id": "alefragnani.Bookmarks",
-      "download": "https://github.com/alefragnani/vscode-bookmarks/releases/download/v13.0.1/Bookmarks-13.0.1.vsix",
-      "version": "13.0.1"
+      "version": "13.2.2",
```

Comment:
I'm not sure about specifying versions, see internal discussions: https://gitpod.slack.com/archives/CQJLAF3SR/p1637060685005600

I think we should rather read version from package.json of cloned git repo or downloaded vsix file.

Reply:
Hm, turned out that some extensions forget to push updated version to GitHub, but only to Marketplace, so in such cases we should provide commit and proper version.